### ISSUE TITLE
Сделать кнопки Продолжить, Остановить, Пауза отдельно для выделенного сеанса и для всех сеансов.

### ISSUE THREAD

**Issue by DmitryShvaika** [1C-Company/1c-edt-issues#323]
### Описание проблемы

Сейчас если в отладку подключены 2 предмета отладки, то нельзя один продолжить, второй оставить стоять на точке останова.

### Описание решения проблемы

Предлагаю сделать дублирующие кнопки "Продолжить", "Остановить" и "Пауза" для выделенного сеанса. Чтобы можно было управлять состоянием отдельного сеанса

**Comment by KovAlexey** [comment]
А это разве не ограничение корявого платформенного отладчика?

**Comment by DmitryShvaika** [comment]
> А это разве не ограничение корявого платформенного отладчика?

Edt разве использует платформенный отладчик? Он же вроде сервер отладки использует...

**Comment by KovAlexey** [comment]
Который является частью платформы

**Comment by KovAlexey** [comment]
Да, я просто не совсем корректно выразился. Речь про сам сервер отладки, который является частью платформы.

**Comment by DitriXNew** [comment]
Или настройте группу запуска и морочьте себе голову :)

**Comment by DmitryShvaika** [comment]
Не понимаю, почему не должно быть  возможности управлять каждым сеансом отдельно
